lib/firehose/server/channel_subscription.rb

```diff
@@ -0,0 +1,103 @@
+module Firehose
+  module Server
+    # Connects to a specific channel on Redis and listens for messages to notify subscribers.
```

Comment:
I think this comment is no longer quite accurate. This connects to a specific channel and listens for a single subscriber. In fact, it either:
* Gets a single batch of messages (if you are behind); or
* Subscribes you to get the next (single) message.

In the latter case, if you have a WebSocket connection, it actually isn't this class that resubscribes you.